fca/src/main/java/org/nmdp/ngs/fca/Concept.java

```diff
@@ -46,10 +41,17 @@ public Concept(final MutableBitSet extent, final MutableBitSet intent) {
         this.extent = extent;
         this.intent = intent;
     }
+    
+    private MutableBitSet and(final Concept that) {
+        return new MutableBitSet().or(this.intent).and(that.intent);
```

Comment:
added ```immutableCopy()/mutableCopy()/unsafeCopy()``` methods to all the bitset implementation classes [here](https://github.com/heuermh/dishevelled/commit/7622a5ad81f71b46bec23539af55eead96c9d849) and deployed a new snapshot version ```3.1-SNAPSHOT```